activeadmin_quill_editor.gemspec

```
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.files         = Dir['{app,lib}/**/*', 'LICENSE.txt', 'Rakefile', 'README.md']
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'activeadmin', '~> 2.0'
+  spec.add_runtime_dependency 'activeadmin', '>= 2.0, < 4'
```

Hey @alejandroperea!
Thank you for your PR.

Can you update the dependency versions syntax?

It should be:

```suggestion
  spec.add_runtime_dependency 'activeadmin', '>= 2.0', '< 4'
```

---

Whoah, good catch 😓 

I've also forgot to update the appraisals. I did another commit for updating them.

---

Yeah, I see your changes only now 😅 

I'm working to update the appraisal config too.
Some configs are outdated.

---

But I'll address them in another PR, your changes looks good 👍